### DISCUSSION TITLE
Código de Autorização para recebimento de devoluções

### DISCUSSION THREAD

**Discussion by Marco-Masa** [rodurma/forum-meli#36] [Envios (Shipments)]
Olá, 
Alguém consegue me ajudar por favor, pois não estou localizando na Api onde consigo consultar o novo Código de Autorização de recebimento de devoluções.
É o código que tem validade diária e que deve ser informado sempre que chegar uma devolução na empresa que fez a venda.
Grato,
Marco Aurélio

**Comment by armendane** [comment]
Cara... eu também pesquisei e nao encontrei... Conseguiu alguma solução?
